src/models/Control.php

```diff
@@ -112,22 +111,93 @@ class Control extends Model {
     await Progressive::genStop();
   }
 
-  public static function importGame(): void {
+  public static async function importGame(): Awaitable<bool> {
+    $files = Utils::getFILES();
+    if ($files->contains('game_file')) {
+      $importer = new JSONImporterController();
+      $result = await $importer->importGame($files['game_file']['tmp_name']);
+      return $result;
+    } else {
+      return false;
+    }
+  }
+
+  public static async function importTeams(): Awaitable<bool> {
+    $files = Utils::getFILES();
+    if ($files->contains('teams_file')) {
+      $importer = new JSONImporterController();
+      $result = await $importer->importTeams($files['teams_file']['tmp_name']);
+      return $result;
+    } else {
+      return false;
+    }
+  }
+
+  public static async function importLogos(): Awaitable<bool> {
+    $files = Utils::getFILES();
+    if ($files->contains('logos_file')) {
+      $importer = new JSONImporterController();
+      $result = await $importer->importLogos($files['logos_file']['tmp_name']);
+      return $result;
+    } else {
+      return false;
+    }
+  }
+
+  public static async function importLevels(): Awaitable<bool> {
+    $files = Utils::getFILES();
+    if ($files->contains('levels_file')) {
+      $importer = new JSONImporterController();
+      $result = await $importer->importLevels($files['levels_file']['tmp_name']);
+      return $result;
+    } else {
+      return false;
+    }
+  }
+
+  public static async function importCategories(): Awaitable<bool> {
+    $files = Utils::getFILES();
+    if ($files->contains('categories_file')) {
+      $importer = new JSONImporterController();
+      $result = await $importer->importCategories($files['categories_file']['tmp_name']);
+      return $result;
+    } else {
+      return false;
+    }
+  }
+
+  public static async function exportGame(): Awaitable<void> {
+    $exporter = new JSONExporterController();
+    $game = await $exporter->genData('game');
+    JSONExporterController::sendJSON($game, 'fbctf_game.json');
+    exit();
+  }
+
+  public static async function exportTeams(): Awaitable<void> {
+    $exporter = new JSONExporterController();
+    $teams = await $exporter->genData('teams');
+    JSONExporterController::sendJSON($teams, 'fbctf_teams.json');
+    exit();
```

Comment:
Can you explain why we need these exit calls? We should just structure the code so they're not needed.